apps/strapi-cms/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "root": true,
+    "extends": [
+      "custom"

```

Comment:
To make this `custom` work we have to define also the `eslint-config` package. I'm going to add it on a separate PR.